pkg/component/gardener/apiserver/deployment.go

```diff
@@ -55,6 +55,10 @@ func (g *gardenerAPIServer) deployment(
 		fmt.Sprintf("--secure-port=%d", port),
 	}
 
+	if g.values.AdminKubeconfigMaxExpiration != nil {
+		args = append(args, fmt.Sprintf("--shoot-admin-kubeconfig-max-expiration=%v", g.values.AdminKubeconfigMaxExpiration.Duration.String()))
```

Comment:
Nit: this could be simplified:
```suggestion
		args = append(args, fmt.Sprintf("--shoot-admin-kubeconfig-max-expiration=%s", g.values.AdminKubeconfigMaxExpiration.Duration))
```